clojure-mode.el

```diff
@@ -352,6 +352,21 @@ Called by `imenu--generic-function'."
               (set-match-data (list def-beg def-end)))))
         (goto-char start)))))
 
+(eval-and-compile
+  (defconst clojure-sym-rest-chars "^][\";\'@\\^`~\(\)\{\}\\"
+    "A black list of chars a clojure symbol must not contain. See
+definiton of 'macros' https://github.com/clojure/clojure/blob/master/src/jvm/clojure/lang/LispReader.java#L88")
```

Comment:
There's a single token on this line, a `{`,.  Perhaps you meant to link to the entire contents enclosed in the braces?  You can highlight lines on github by clicking a line number and holding <kbd>shift</kbd> while clicking a second line number.